tobac/utils.py

```diff
@@ -556,57 +558,53 @@ def add_coordinates(t, variable_cube):
         elif variable_cube.coord(coord).ndim == 2:
 
             if variable_cube.coord_dims(coord) == (hdim_1, hdim_2):
-                f = interp2d(dimvec_2, dimvec_1, variable_cube.coord(coord).points)
-                coordinate_points = np.asarray(
-                    [f(a, b) for a, b in zip(t["hdim_2"], t["hdim_1"])]
-                )
-
-            if variable_cube.coord_dims(coord) == (hdim_2, hdim_1):
-                f = interp2d(dimvec_1, dimvec_2, variable_cube.coord(coord).points)
-                coordinate_points = np.asarray(
-                    [f(a, b) for a, b in zip(t["hdim_1"], t["hdim_2"])]
-                )
```

Comment:
**issue**: In the case that `variable_cube.coord_dims(coord) == (hdim_2, hdim_1)` no interpolation will be done.

**suggestion**: This case could be handled analogously to  `variable_cube.coord_dims(coord) == (hdim_1, hdim_2)`